build.sbt

```diff
@@ -47,16 +48,17 @@ lazy val meta = Project("scalan-meta", file("meta"))
     fork in run := true)
 
 lazy val core = Project("scalan-core", file("core"))
-  .dependsOn(common % allConfigDependency)
+  .dependsOn(common % allConfigDependency, meta % allConfigDependency)
   .settings(commonSettings,
     libraryDependencies ++= Seq(
       "com.chuusai" %% "shapeless" % "2.2.4",
       "cglib" % "cglib" % "3.1",
-      "org.objenesis" % "objenesis" % "2.1"))
+      "org.objenesis" % "objenesis" % "2.1"),
+      fork in Test := true)
```

Comment:
This also requires all dependent projects to set this. Coincidentally, me, @alexanderfilippov and @PCMNN just spent quite a bit of time and effort figuring this out :( #105